playground/SqlServerEndToEnd/SqlServerEndToEnd.AppHost/sql1.module.bicep

```diff
@@ -1,49 +1,43 @@
-targetScope = 'resourceGroup'
-
-@description('')
+@description('The location for the resource(s) to be deployed.')
 param location string = resourceGroup().location
 
-@description('')
 param principalId string
 
-@description('')
 param principalName string
 
-
-resource sqlServer_x8iP8H24Z 'Microsoft.Sql/servers@2020-11-01-preview' = {
-  name: toLower(take('sql1${uniqueString(resourceGroup().id)}', 24))
-  location: location
-  tags: {
-    'aspire-resource-name': 'sql1'
-  }
-  properties: {
-    version: '12.0'
-    publicNetworkAccess: 'Enabled'
-    administrators: {
-      administratorType: 'ActiveDirectory'
-      login: principalName
-      sid: principalId
-      tenantId: subscription().tenantId
-      azureADOnlyAuthentication: true
+resource sql1 'Microsoft.Sql/servers@2021-11-01' = {
+    name: take('sql1-${uniqueString(resourceGroup().id)}', 63)
+    location: location
+    properties: {
+        administrators: {
+            administratorType: 'ActiveDirectory'
```

Comment:
I think we should switch over to using AAD administrators, this is the old way of specifying AAD access. Its caused some problems but we couldn't do anything about it until now.

Reply:
https://learn.microsoft.com/en-us/azure/templates/microsoft.sql/servers/administrators?pivots=deployment-language-bicep#resource-format

Reply:
Related: https://github.com/dotnet/aspire/issues/3101

Reply:
That's not this change. This change is just updating the versions. Its goal is to not change behavior, if possible.